nodemon.json

```diff
@@ -2,7 +2,7 @@
   "env": {
     "NODE_ENV": "development"
   },
-  "ignore": ["**/*.test.ts", "**/*.spec.ts", ".git", "node_modules", "src"],
+  "ignore": ["**/*.test.ts", "**/*.spec.ts", ".git", "node_modules"],
```

Comment:
Can you add back "src" ?

Reply:
done, do I put my branch on staging or merge directly on master?

Reply:
@DavidRobertAnsart

Reply:
On master directly. Then the search-bar will automatically be deployed on prod.